ethicml/algorithms/inprocess/logistic_regression.py

```diff
@@ -30,6 +30,9 @@ class LR(InAlgorithm):
     seed: int = 888
     is_fairness_algo = False
 
+    def __post_init__(self):
+        self._hyperparameters = {"C": self.C}
```

Comment:
Would have preferred to do this with a property instead of with `__post_init__` but I don't see an elegant way to do it here.